src/views/diff/Diff.vue

```diff
@@ -1,168 +0,0 @@
-<template>
-  <div class="fill-height canvas">
-    <SideNavigation :app="true" />
-    <v-container fluid class="pa-4">
-      <v-row v-if="loading" no-gutters>
-        <v-col cols="6" class="pr-1">
-          <v-skeleton-loader v-if="loading" type="card-heading, image"></v-skeleton-loader>
-        </v-col>
-        <v-col cols="6">
-          <v-skeleton-loader v-if="loading" type="card-heading, image"></v-skeleton-loader>
-        </v-col>
-      </v-row>
-      <v-row v-if="!loading && res" no-gutters>
-        <v-col cols="12">
-          <template v-if="isAuthorized">
```

Comment:
Only auth section should be removed, not the entire diff viewer

Reply:
@m-kus my bad, returned it back.